### PR TITLE
feat: use dynamic chromedriver versions

### PR DESCRIPTION
**Pull request by loc**
Instead of being tied to the chromedriver version that ships with Old Joe, let's download the right version dynamically depending on the version of Electron we're testing.

### CODE REVIEW

File: src/renderer/driver.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import * as path from 'path';
 import { remote } from 'electron';
```
Comment (erickzhao) [inline src/renderer/driver.ts]: Above this line: `path` is no longer used so we can remove the import.

Comment (loc) [reply src/renderer/driver.ts]: 🦅👁

**Comment by loc** [comment]
> Going forward, we should probably think about how we want to version our releases such that we can make Old Joe vs. Slack Client compatibility evident.

Won't all Old Joe versions be compatible with all Slack clients going forward?

**Comment by erickzhao** [comment]
@loc In terms of Electron versions, but I was wondering along the lines of big webapp changes that break our smoke tests. Will upgrades like those be more trivial? 🤔

**Pull request merged**